source/globals.d.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/consistent-indexed-object-style */
```

Comment:
Had to do this because all the types would get messed up

Reply:
I think its https://github.com/typescript-eslint/typescript-eslint/pull/2678 which is not released yet

Reply:
Let's wait for it